### ISSUE TITLE
should report missing space

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1371]
apparently both ABAP and abaplint is okay with this, but it just looks strange

```abap
    IF token-str EQ'RETURNING'.
      has_returning_parameter = abap_true.
    ENDIF.
```
